lib/Analizo/Command/metrics.pm

```diff
@@ -33,6 +33,7 @@ sub opt_spec {
     [ 'includedirs|I=s',  'include <dirs> (a colon-separated list of directories) with C/C++ header files', { default => '.' } ],
     [ 'libdirs|L=s',  'include <dirs> (a colon-separated list of directories) with C/C++ static and dynamic libraries files', { default => '.' } ],
     [ 'libs=s',  'include <dirs> (a colon-separated list of directories) with C/C++ linked libraries files', { default => '.' } ],
+    [ 'nocache|noc', 'run metrics without use the cache' ],
```

Comment:
would be interesting if `nocache` works for other analizo tools, not only for the `metrics`, for exemple for the `graph` tool, it's very interesting that the `nocache` works for that tool also, the same for other tools like: `dsm`, `metrics_batch, etc...